src/components/React/ColorMappingController.js

```diff
@@ -1,73 +1,31 @@
 import React, { Component } from "react";
 import ReactModal from "react-modal";
+// import { DataTable } from "primereact/datatable";
+// import { Column } from "primereact/column";
```

Comment:
delete commented out code

Reply:
I left it in just in case we wanted to keep using the modal. Do you think the dropdown is definitely the way to go? If so I'll delete it

Reply:
Yes, I think dropdown is the way to go.  And even if we change our minds, git has the whole history and we know we can go find it, so either way the commented out code should be deleted.